libs/application-generic/src/usecases/select-integration/select-integration.spec.ts

```diff
@@ -117,7 +117,10 @@ describe('select integration', function () {
           new ExecutionLogQueueService(new WorkflowInMemoryProviderService()),
           new GetFeatureFlag(new FeatureFlagsService())
         ),
-        new CompileTemplate()
+        // @ts-ignore
+        new CompileTemplate(),
+        // @ts-ignore
+        new NormalizeVariables()
       ),
```

Comment:
If it's really necessary to override type safety, prefer to use `ts-expect-error` with a trailing `- <reason>` instead of `ts-ignore`. `ts-expect-error` will fail if the types are fixed, whilst `ts-ignore` will not. This helps us to keep on top of type-safety.

Preferably, can we fix the types here? If not possible, replace with `ts-expect-error`

```suggestion
        // @ts-expect-error - <insert reason here>
        new CompileTemplate(),
        // @ts-expect-error - <insert reason here>
        new NormalizeVariables()
      ),
```

Reply:
thanks, will fix this one